src/PaypalCountryCodeMatrice.php

```diff
@@ -64,10 +64,10 @@ public function getPrestashopIsoCode($isoCode)
             return false;
         }
 
-        if (false === array_search($isoCode, self::MATCH_ISO_CODE)) {
+        if (!in_array($isoCode, self::MATCH_ISO_CODE)) {
             return $isoCode;
         }
 
-        return array_search($isoCode, self::MATCH_ISO_CODE);
+        return in_array($isoCode, self::MATCH_ISO_CODE);
```

Comment:
Something is off here, I think, because if the purpose is that you return the mapped value if it exists in the static then it will not do that after your change if I'm not wrong.

```
public function getPrestashopIsoCode($isoCode)
{
    if (!is_string($isoCode)) {
        return false;
    }

    $mappedCode = array_search($isoCode, self::MATCH_ISO_CODE);
    if ($mappedCode !== false) {
        return $mappedCode;
    }

    return $isoCode;
}
```